security_rules/security-rules.json

```diff
@@ -58,6 +58,7 @@
             ".validate": "newData.isString() && newData.val().matches(/^[-a-z0-9]{3,20}$/) || newData.val() == null"
           },
           "school": {
+            ".write": "newData.val() != null && (auth != null && auth.uid == $userId)",
```

Comment:
It shouldn't be necessary. The parent rule is identical and grants creating/editing that field.